www/content/workstation/install_workstation.md

```diff
@@ -133,6 +133,18 @@ Alternately, install Chef Workstation using Homebrew:
 3.  **Optional:** Set the default shell. On Microsoft Windows it is
     strongly recommended to use Windows PowerShell instead of `cmd.exe`.
 
+#### Headless Unattended Install
+
+To install Chef Workstation on a headless Windows system and
```

Comment:
Let's add a definition of headless, just to avoid the remote possibility of confusion.

```suggestion

"Headless" systems are configured to operate without a monitor (the "head") keyboard, and mouse.  They are usually controlled over a network connection.

To install Chef Workstation on a headless Windows system, 
```